requirements.txt

```
@@ -1,5 +1,6 @@
 # Our libraries
 spacy-legacy>=3.0.7,<3.1.0
+spacy-loggers>=1.0,<2.0
```

Just for consistency:

```suggestion
spacy-loggers>=1.0.0,<2.0.0
```